lib/tty/logger/handlers/console.rb

```diff
@@ -150,9 +150,12 @@ def format_backtrace(event)
         #
         # @api private
         def configure_styles(event)
-          style = STYLES.fetch(event.metadata[:name]&.to_sym, {}).dup
-          (@styles[event.metadata[:name]&.to_sym] || {}).each do |k, v|
-            style[k] = v
+          style = if event.metadata[:name]
+            STYLES.fetch(event.metadata[:name].to_sym, {})
+              .dup
+              .merge!(@styles[event.metadata[:name].to_sym] || {})
+          else
+            {}
```

Comment:
I like the idea of chaining the transformation. How about we drop the `style` variable all together? 

```ruby
return {} if event.metadata[:name].nil?

STYLES.fetch.... 
```